app/views/guests/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1>Guest Information</h1>
+<h3><%= "#{@guest.name}" %></h3>
+<% @rooms.each do |room| %>
+  <p><%= "Suite: #{room.suite}" %></p>
+  <p><%= "Rate: #{room.rate}" %></p>
+  <p><%= "Hotel: #{room.hotel.name}" %></p>
+<% end %>
+
+<h2>Add a Room</h2>
+<%= form_with url: "/guests/#{@guest.id}/rooms", data: { turbo: false }, local: true do |f| %>
+  <%= f.label "Enter room id:" %>
+  <%= f.text_field :room_id %>
+  <%= f.submit "Submit" %>
+<% end %>
```

Comment:
I'm guessing forms are `POST` by default so it doesn't have to be specified? In any case, the route is perfect. And checking the `routes.rb` file I see that it's making a post, so great job.